src/app/content/studyGuides/components/StudyGuidesListElement.tsx

```diff
@@ -35,6 +35,7 @@ const HighlightAnnotation = styled.div`
   display: flex;
   padding: 1.2rem 0;
   color: ${theme.color.text.black};
+  white-space: pre-wrap;
```

Comment:
In MH we are also using:
```
word-break: break-word;
overflow-wrap: break-word;
```
Should we add it here to make it consistent?

Reply:
I thought about it but it felt little unrelated.

Pretty sure `overflow-wrap: break-word` is obsolete when `word-break` is set to `break-word`. I think its only relevant if a single word would span over the width of the container; it seems to break URLs without it. 

I've added the `word-break`